cli/src/commands/browse.ts

```diff
@@ -0,0 +1,28 @@
+import Command from '../utils/command';
+import terminalLink from 'terminal-link';
+
+export default class Browse extends Command {
+  public static description = 'list templates';
+  public static hidden = false;
+  public static examples = [
```

Comment:
Redundant, as the output for EXAMPLES becomes the same as USAGE? (similar for other `examples` for commands with no arguments)

Reply:
keeping it for now to be consistent with other commands. once we decide on examples in general we can wipe everything out.